docs/contribute.md

```diff
@@ -124,3 +124,32 @@ For a specific task
 ```bash
 node make.js testLegacy --task Xcode
 ```
+
+# Package
+
+After a local code update and testing, you should test any changes in your Azure DevOps organization before submitting a pull request. 
+
+## Prepare your organization
+In order to upload an extension directly to your Azure DevOps organization you must be an [administrator](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?view=vsts#security) of "All agent pools" or have a PAT token associated with an administrator.  
+
+## Update metadata (optional)
```

Comment:
I'd remove the optional tag here - if people don't do this we risk weird conflicts, and even though its technically optional I don't really want to call attention to that. I usually read optional as "skip this step unless you have a reason not to"